Astronomy/Utilities/NetworkDataLoader.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  NetworkDataLoader.swift
+//  Astronomy
+//
+//  Created by Cora Jacobson on 9/30/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+protocol NetworkDataLoader {
+    
+    func loadData(from request: URLRequest, completion: @escaping (Data?, Error?) -> Void)
+    
+    func loadData(from url: URL, completion: @escaping (Data?, Error?) -> Void)
```

Comment:
I know this was in the instructions this way, but I would suggest adding back in the URLResponse in these methods. I find it very helpful to have the response to check when things go wrong, and it's honestly not much more work.